ctru-rs/src/services/cam.rs

```diff
@@ -783,7 +974,30 @@ impl Cam {
         }
     }
 
-    /// Plays the specified sound based on the [ShutterSound] argument
+    /// Play the specified sound based on the [`ShutterSound`] argument
+    ///
+    /// # Notes
+    ///
+    /// Playing the shutter sound does not require a living handle to the [`Ndsp`](crate::services::ndsp::Ndsp) service.
+    /// Volume will always be maxed out to ensure everyone within photo range can hear the picture being taken (as by japanese law).
```

Comment:
Oops, this one ended up on the wrong line somehow, just a quick typo fix:
```suggestion
    /// Volume will always be maxed out to ensure everyone within photo range can hear the picture being taken (as by Japanese law).
```